src/test/java/com/mongodb/kafka/connect/source/producer/SchemaAndValueProducerTest.java

```diff
@@ -280,7 +280,7 @@ void testBsonSchemaAndValueProducer() {
         "Assert schema and value matches",
         () -> assertEquals(Schema.BYTES_SCHEMA.schema(), actual.schema()),
         // Ensure the data length is truncated.
-        () -> assertEquals(1071, ((byte[]) actual.value()).length),
+        () -> assertEquals(1712, ((byte[]) actual.value()).length),
```

Comment:
@rozza I have no idea where the previous `1071` value came from, or what this assertion is. I put `1712` as this is the new value we have in this test as a result of changes. Should we just delete the assertion as it is useless, given that a reader cannot understand what this is, and doing what I did appears to be the only way to make schema changes, unless the reader has access to arcane knowledge?

Reply:
It is expected that the serialized value is longer, as it has more data now. But I know neither where the asserted value is supposed to come from, nor why the serialized value is expected to be truncated.

Reply:
We should not delete the assertion, because it guards against unexpected changes to the value, even though we do not know exactly how to compute the value.

Reply:
iirc it was added to ensure that the size is as expected - it really is testing documentToByteArray and its handling of RawBsonDocuments. So I agree it can be removed from here and is better suited to its own test and / or is already tested in BsonValueToSchemaAndValueTest

Reply:
Removed the assertions.